src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -215,7 +215,7 @@ const AnalysisCard = ({
     getFileName(name)
   ])
 
-  const toolIconSrc: string = Utils.switchCase(currentRuntimeToolLabel,
+  const toolIconSrc: string = Utils.switchCase(application,
```

Comment:
This is an appropriate fix!

However.......
Let's look at what currentRuntimeToolLabel is. It's the correct type for this switch case. Ideally we'd want to be using that instead of application (which is of type any).
How was it derived? You would think it's a property of the runtime. No, it's a LABEL.
`export const getToolLabelFromRuntime = (runtime: Runtime): ToolLabel => runtime?.labels?.tool`
This label isn't even guaranteed!! And the type is of 'string' being cast to ToolLabel.

This is one of the reasons why we need to deprecate using labels for front end logic that isn't specific to the "labels" feature (which I don't think was ever used?)

Reply:
wow this got me `runtime?.labels?.tool`

Reply:
Yeahhh..........